Package.swift

```diff
@@ -86,11 +87,16 @@ let package = Package(
                 "Bookmarks"
             ]
         ),
-         .target(
-            name: "BloomFilterWrapper",
+        .target(
+            name: "BloomFilterObjC",
```

Comment:
The existing filter was renamed to ObjC.  This should ideally be kept private, as importing this directly would recreate the issue we had.